tests/test_standalone_stac_item.py

```diff
@@ -21,35 +20,11 @@ def quote_none_safe(url):
 
 @pytest.mark.online
 def test_standalone_stac_item_thredds_ncml():
-    thredds_url = "https://pavics.ouranos.ca/twitcher/ows/proxy/thredds"
-    thredds_path = "birdhouse/testdata/xclim/cmip6"
-    thredds_nc = "sic_SImon_CCCma-CanESM5_ssp245_r13i1p2f1_2020.nc"
-    thredds_catalog = f"{thredds_url}/catalog/{thredds_path}/catalog.html"
-    thredds_ds = f"{thredds_path}/{thredds_nc}"
-    thredds_ncml_url = (
-        f"{thredds_url}/ncml/{thredds_path}/{thredds_nc}"
-        f"?catalog={quote_none_safe(thredds_catalog)}&dataset={quote_none_safe(thredds_ds)}"
-    )
-
-    # FIXME: avoid hackish workarounds
-    data = requests.get(thredds_ncml_url).text
-    attrs = xncml.Dataset.from_text(data).to_cf_dict()
-    attrs["access_urls"] = {  # FIXME: all following should be automatically added, but they are not!
-        "HTTPServer": f"{thredds_url}/fileServer/{thredds_path}/{thredds_nc}",
-        "OPENDAP": f"{thredds_url}/dodsC/{thredds_path}/{thredds_nc}",
-        "WCS": f"{thredds_url}/wcs/{thredds_path}/{thredds_nc}?service=WCS&version=1.0.0&request=GetCapabilities",
-        "WMS": f"{thredds_url}/wms/{thredds_path}/{thredds_nc}?service=WMS&version=1.3.0&request=GetCapabilities",
-        "NetcdfSubset": f"{thredds_url}/ncss/{thredds_path}/{thredds_nc}/dataset.html",
-    }
-
+    url = "https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/birdhouse/testdata/xclim/cmip6/sic_SImon_CCCma-CanESM5_ssp245_r13i1p2f1_2020.nc"
+    attrs = ncattrs(url)
     stac_item_id = CMIP6populator.make_cmip6_item_id(attrs["attributes"])
     stac_item = STAC_item_from_metadata(stac_item_id, attrs, CMIP6ItemProperties, GeoJSONPolygon)
-
-    ref_file = os.path.join(CUR_DIR, "data/stac_item_testdata_xclim_cmip6_ncml.json")
-    with open(ref_file, mode="r", encoding="utf-8") as ff:
-        reference = json.load(ff)
-
-    assert stac_item.to_dict() == reference
+    assert stac_item.validate()
```

Comment:
I think that might not be validating all the contents, only the CMIP6 schema. 

That would be because `CMIP6ItemProperties` is based on `pydantic` schema, while `pystac.Item` use their own `pystac.validation.STACValidator`.

Reply:
In theory, `validate()` returns the schemas it used to validate the content. Here it returns the STAC item schema. 
It validates only the STAC item for now (no extensions).

Reply:
validate calls the pystac validator based on the JSONSchema. The test is more explicit about this now.

Reply:
Function `STAC_item_from_metadata` returns a `STACpopulator.stac_utils.STACItem` derived from `pydantic.BaseModel`.

Unless it does something special, I don't think that would call the `pystac` validator. It would if the object returned was `pystac.Item`. 

Note, I'm trying to see if we can get rid of this dual validator approach in https://github.com/crim-ca/stac-populator/pull/32

Reply:
Argh, my bad. I thought we had a pystac.Item object...

Reply:
In fact we do !  `STAC_item_from_metadata` returns a pystac.Item:
```
item = pystac.Item(**json.loads(item.model_dump_json(by_alias=True)))
```